src/ifcopenshell-python/ifcopenshell/file.py

```diff
@@ -464,6 +466,16 @@ def schema_version(self) -> tuple[int, int, int, int]:
             number = re.search(prefix + r"(\d)", schema)
             version.append(int(number.group(1)) if number else 0)
         return tuple(version)
+        
+    @property 
+    def mvd(self):
+        if not LARK_AVAILABLE:
+            return None
+        file_description = self.wrapped_data.header.file_description
+        return MvdInfo(
+            get_description=lambda: file_description.description, 
+            set_description=lambda d: setattr(file_description, "description", tuple(d))
```

Comment:
I guess the more conventional way would be to simply pass

```suggestion
            self.header
```

And in your tests (if needed) you create a mock datastructure that mimics the layout of the file.header object

Because you don't really need a more general interface like this, get_description/set_description are encapsulated in self.header

Reply:
That makes sense, I didn't think about offloading this to the MVDInfo class. Just needed to do a couple of tweaks in the mvd parser and modifying some tests.
e.g. 
```
    @property
    def description(self) -> list[str]:
        return self._get_description()
```
must be changed to 
```
    @property
    def description(self) -> list[str]:
        return self._header.file_description.description
```
https://github.com/IfcOpenShell/IfcOpenShell/pull/6154/commits/8983b0571b0b189d0ff1122b291f81bd25d4ebc6